src/components/TodoForm/TodoForm.scss

```diff
@@ -0,0 +1,3 @@
+.error {
+  color: red;
+}
```

Comment:
Add [empty line in the end of file](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)